security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,22 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.11.4"]
+== Release notes for {cert-manager-operator} 1.11.4
+
+Issued: 2023-07-26
+
+The following advisory is available for the {cert-manager-operator} 1.11.4:
+
+* link:https://access.redhat.com/errata/RHEA-2023:4081[RHEA-2023:4081]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs).
+
+[id="cert-manager-operator-1.11.4-bug-fixes"]
+=== Bug fixes
+
+* Previously, the {cert-manager-operator} did not allow installing older versions of the {cert-manager-operator}. Now, you can install older versions of the {cert-manager-operator} by selecting the *Update approval* strategy as *Manual* in the web console in {product-title} 4.14. You can install older versions of the {cert-manager-operator} by using the command-line interface (CLI) in {product-title} 4.12 and {product-title} 4.13. For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-install.adoc#cert-manager-operator-install[Installing the {cert-manager-operator}]. (link:https://issues.redhat.com/browse/OCPBUGS-16393[*OCPBUGS-16393*])
```

Comment:
```suggestion
* Previously, the {cert-manager-operator} did not allow you to install older versions of the {cert-manager-operator}. Now, you can install older versions of the {cert-manager-operator} using the web console or the command-line interface (CLI).  For more information on how to use the web console to install older versions, see xref:../../security/cert_manager_operator/cert-manager-operator-install.adoc#cert-manager-operator-install[Installing the {cert-manager-operator}]. (link:https://issues.redhat.com/browse/OCPBUGS-16393[*OCPBUGS-16393*])
```

Reply:
The instructions are a duplication of what can be found in the xref so I don't think they are necessary. I also think we should be careful with mentioning 4.14 here and looking at the docs you can already do this in 4.12+. 

I wasn't sure if there was a specific reason why CLI on 4.12 and 4.13 was being mentioned. If this is getting merged back to 4.12 I thought this was implied. 

My goal in the suggestion was to condense and shorten things. I hope this helps and understand if they versions are necessary for technical accuracy.

Reply:
Agreed. Addressed